cli/wadltest/src/test/scala/com/rackspace/com/papi/components/checker/cli/WadlTestSuite.scala

```diff
@@ -53,58 +50,38 @@ class WadlTestSuite extends FunSuite {
   }
 
 
-  test ("--help should generate usage info") {
-    WadlTest.parser.reset()
-    intercept[ArgotUsageException] {
-      WadlTest.handleArgs(Array("--help"))
-    }
+  test("-bad_data should generate usage info") {
+    WadlTest.handleArgs(Array("-bad_data")) shouldBe None
```

Comment:
This does make for an easier way to test, but the return is kind of big and bulky.
Maybe it should return a case class?